### PR TITLE
sigmoid added to jax frontend

### PR DESCRIPTION
**Pull request by VikramxD**

### CODE REVIEW

**Comment by VikramxD** [comment]
pr for #2963 from todolist #1515

1. Implemented sigmoid in  ivy/functional/frontends/jax/nn/non_linear_activations.py
2. added tests for sigmoid ivy_tests/test_ivy/test_frontends/test_jax/test_activations.py
3.Following specifications described in https://jax.readthedocs.io/en/latest/_autosummary/jax.nn.sigmoid.html?highlight=jax%20sigmoid

**Comment by VikramxD** [comment]
@bicycleman15 can you review my pull request

**Comment by bicycleman15** [comment]
@jkeane508 Could you take this one ?

**Comment by VikramxD** [comment]
@jkeane508 can you review my pull request?

**Review by jkeane508 — APPROVED** [review]
Looks good, just to need to fix some conflicts, will merge after!
